.travis.yml

```
@@ -42,4 +42,5 @@ matrix:
       env: CASK=puppet-bolt
     - osx_image: xcode10.2
       env: FORMULA=wash
+      env: FORMULA=relay
```

This needs to be a new item.
```suggestion
    - osx_image: xcode10.2
      env: FORMULA=relay
```